CHANGELOG

```diff
@@ -5,8 +5,10 @@ v3.17 (Month 2020)
   * Card improvements:
     - Activity Feed now shows board name and link
     - No mandatory due date
-  * Comments can now have Textile markup
-  * Comments now show author's name instead of email
+  * Comments
+    - Added Textile markup
+    - Uncommitted changes for comments now persists in case it wasn't saved
```

Comment:
There is something wrong with this sentence structure, can you please reword in a way that makes sense to the user? "Uncommitted changes" is developer speak!